pkg/op/config.go

```diff
@@ -47,6 +47,7 @@ type Configuration interface {
 	RequestObjectSupported() bool
 	RequestObjectSigningAlgorithmsSupported() []string
 
+	SupportedClaims() []string
```

Comment:
Adding a method to the interface would be a breaking change, instead we should use the global function.